WebApp/autoreduce_webapp/templates/help.html

```diff
@@ -386,6 +386,21 @@ <h4>Where does my reduced data go?</h4>
                     </p>
                 </div>
             </section>
+            <section class="help-topic panel panel-default" data-category="question">
```

Comment:
You will need a `data-topics="..."` tag here which will make the new topic searchable using a set of keywords. See the other topics for examples.

Reply:
I deliberately didn't add this as I wasn't sure if you were still going to use them when you implement the new search. But I can put them in sure

Reply:
Thanks, I think for now we should keep the keywords static but in the future the keywords will be generated from the topic description.